pkg/apis/core/types.go

```diff
@@ -2378,6 +2379,12 @@ type PodAffinityTerm struct {
 	// selected pods is running.
 	// Empty topologyKey is not allowed.
 	TopologyKey string
+	// Selects Namespaces using cluster scoped-labels.  This
+	// matches namespaces selected by this label selector.
+	// This field follows standard label selector semantics.
+	// If present but empty, this selector selects all namespaces.
```

Comment:
Please add to the comment that
"if this selector is not provide (nil), the Pod's namespace will be used."